src/openapi/streaming/streaming.spec.ts

```diff
@@ -81,24 +102,34 @@ describe('openapi Streaming', () => {
     afterEach(() => uninstallClock());
 
     function mockSubscription() {
-        function changeState(state) {
-            this.stateChangedCallback.forEach((callback) => callback(state));
-        }
+        type Callback = (state: SubscriptionState) => void;
+
+        // function changeState(state) {

```

Comment:
Why this is being commented?

Reply:
it was used in lin 132 but the changeState property from line 132 was not used anywhere so I temporarily commented it out but forgot to remove :) Will remove it